Gemfile.jruby-2.5.lock.release

```diff
@@ -92,12 +92,12 @@ GEM
     elastic-app-search (7.8.0)
       jwt (>= 1.5, < 3.0)
     elastic-workplace-search (0.4.1)
-    elasticsearch (7.15.0)
-      elasticsearch-api (= 7.15.0)
-      elasticsearch-transport (= 7.15.0)
-    elasticsearch-api (7.15.0)
+    elasticsearch (7.16.1)
+      elasticsearch-api (= 7.16.1)
```

Comment:
I don't think we need to change the ES client, let's just keep the changes to the affected plugins